src/vs/editor/contrib/suggest/browser/suggestWidget.ts

```diff
@@ -559,9 +561,13 @@ export class SuggestWidget implements IDisposable {
 			this._onDidSelect.resume();
 		}
 
-		this._layout(this.element.size);
-		// Reset focus border
-		this._details.widget.domNode.classList.remove('focused');
+		this._pendingLayout?.dispose();
```

Comment:
The pending layout (and details showing) should also be cancelled when hiding the widgets via `hideWidget` and `toggleDetails`